crates/autopilot/src/database.rs

```diff
@@ -148,8 +164,10 @@ mod tests {
     #[tokio::test]
     #[ignore]
     async fn postgres_count_rows_in_table_() {
-        let db = Postgres::new("postgresql://").await.unwrap();
-        let mut ex = db.0.begin().await.unwrap();
+        let db = Postgres::new("postgresql://", NonZeroUsize::new(500).unwrap())
```

Comment:
For testing purposes it will probably nicer long term if we have a constructor `with_defaults()`. That way we don't have to touch all the e2e tests in case we extend the config in the future.